docs/ruleset_functions/is_zone_mechanically_heated_and_not_cooled.md

```diff
@@ -0,0 +1,34 @@
+# is_zone_mechanically_heated_and_not_cooled
+**Schema Version:** 0.0.23  
+
+**Description:** determines whether a zone is mechanically heated, but not cooled.  DOES NOT CHECK FOR TRANSFER AIR
+
```

Comment:
@KarenWGard I assume this is to check for the heated-only system  requirement (exception e), correct?

Reply:
yes